src/main/java/net/mrscauthd/beyond_earth/events/OverlayEvents.java

```diff
@@ -125,61 +123,58 @@ public static void Overlay(RenderGameOverlayEvent.PostLayer event) {
                 if (entity.getVehicle() instanceof RocketTier1Entity) {
                     timer = entity.getVehicle().getEntityData().get(RocketTier1Entity.START_TIMER);
                 }
-
-                if (entity.getVehicle() instanceof RocketTier2Entity) {
+                else if (entity.getVehicle() instanceof RocketTier2Entity) {
                     timer = entity.getVehicle().getEntityData().get(RocketTier2Entity.START_TIMER);
                 }
-
-                if (entity.getVehicle() instanceof RocketTier3Entity) {
+                else if (entity.getVehicle() instanceof RocketTier3Entity) {
                     timer = entity.getVehicle().getEntityData().get(RocketTier3Entity.START_TIMER);
                 }
-
-                if (entity.getVehicle() instanceof RocketTier4Entity) {
+                else if (entity.getVehicle() instanceof RocketTier4Entity) {
                     timer = entity.getVehicle().getEntityData().get(RocketTier4Entity.START_TIMER);
                 }
 
-                int width = event.getWindow().getGuiScaledWidth() / 2 - 31;
-                int high = event.getWindow().getGuiScaledHeight() / 2 / 2;
+                final int width = event.getWindow().getGuiScaledWidth() / 2 - 31;
+                final int high = event.getWindow().getGuiScaledHeight() / 2 / 2;
 
                 Gui mc = Minecraft.getInstance().gui;
 
                 if (timer > -1 && timer < 20) {
                     RenderSystem.setShaderTexture(0, new ResourceLocation(BeyondEarthMod.MODID, "textures/timer/timer10.png"));
                     mc.blit(event.getMatrixStack(), width, high, 0, 0, 60, 38, 60, 38);
                 }
-                if (timer > 20 && timer < 40) {
+                else if (timer > 20 && timer < 40) {
```

Comment:
timer cannot simultaneously be less than 20 and more than 20, so use `else if` instead of `if`